app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    sort_command = command.split()
```

Comment:
Use unpacking after the length check here to avoid using indexes and be more descriptive